run-build-functions.sh

```diff
@@ -408,33 +407,39 @@ install_dependencies() {
   fi
 
   # Swift Version
-  : ${SWIFT_VERSION="$defaultSwiftVersion"}
+  defaultSwiftVersion="5.2"
   if [ -f .swift-version ]
   then
     SWIFT_VERSION=$(cat .swift-version)
     echo "Attempting Swift version '$SWIFT_VERSION' from .swift-version"
   fi
 
-  swiftenv global ${SWIFT_VERSION} > /dev/null 2>&1
-  export CUSTOM_SWIFT=$?
-
-  if [ -d $NETLIFY_CACHE_DIR/swift_version/$SWIFT_VERSION ]
+  # If Package.swift is present and no Swift version is set, use a default
+  if [ -f Package.swift ]
   then
-    echo "Started restoring cached Swift version"
-    rm -rf $SWIFTENV_ROOT/versions/$SWIFT_VERSION
-    cp -p -r $NETLIFY_CACHE_DIR/swift_version/${SWIFT_VERSION} $SWIFTENV_ROOT/versions/
-    swiftenv rehash
-    echo "Finished restoring cached Swift version"
+    : ${SWIFT_VERSION="$defaultSwiftVersion"}
```

Comment:
Would it be better to make this a `else if` from the `if [ -f .swift-version ]` block?
This would also remove the need for `: ${...=...}`.

Reply:
I believe we would still need the conditional override because `SWIFT_VERSION` could be set by the user.

Reply:
Oh yes, good point.